src/main/osd/osd.c

```diff
@@ -359,9 +359,6 @@ void osdInit(displayPort_t *osdDisplayPortToUse)
     cmsDisplayPortRegister(osdDisplayPort);
 #endif
 
-    backgroundLayerSupported = displayLayerSupported(osdDisplayPort, DISPLAYPORT_LAYER_BACKGROUND);
```

Comment:
How did this change get in here? I think this is an accidental revert of an unrelated change. @etracer65 can you confirm?

Reply:
Yes, this is an accidental revert. Good catch as it wouldn’t be noticed except for the loss in performance optimization for max7456.

Reply:
Doh! Good catch!